service/src/test/java/bio/terra/workspace/app/controller/CreateAzureStorageContainerSasTokenTest.java

```diff
@@ -32,11 +32,11 @@ public class CreateAzureStorageContainerSasTokenTest extends BaseAzureUnitTest {
   private UUID workspaceId;
   private UUID storageContainerId;
 
-  ArgumentCaptor<SasTokenOptions> sasTokenOptionsCaptor =
+  final ArgumentCaptor<SasTokenOptions> sasTokenOptionsCaptor =
```

Comment:
(here and elsewhere) I don't think we really get any value out of adding `final` to all of these test variables. Since people tend to just remove `final` if we need to modify the variable in a new test, it mostly just signals "this isn't being modified right now", not "this should never be modified".

Reply:
was part of addressing spot bug warnings. agree no value add
will remove

Reply:
Left it as is for class level variables (instance / static)
no final spec for local (function variables)